### PR TITLE
[enterprise-4.12] OSDOCS-4137: add SE RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.12

Issue: https://issues.redhat.com/browse/OSDOCS-4137

Link to docs preview: http://file.rdu.redhat.com/jdohmann/OSDOCS-4118-RN/release_notes/ocp-4-12-release-notes.html#ocp-4-12-secure-execution-z-linux-one

QE review:
- [x] QE has approved this change.

Additional information: build is failing on the xref to the feature doc https://github.com/openshift/openshift-docs/pull/52661 but should be fine when that doc lands.

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -44,6 +44,14 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-12-secure-execution-z-linux-one"]
```
Comment (SNiemann15) [inline release_notes/ocp-4-12-release-notes.adoc:47]: @jldohmann 
Please change entry as follows: 
```
==== IBM Secure Execution on {ibmzProductName} and LinuxONE (Technology Preview)
{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and LinuxONE (s390x architecture) as a Technology Preview feature. IBM Secure Execution provides the highest level of isolation and security for cluster workloads, and you can enable it by using an IBM Secure Execution-ready QCOW2 boot image.

To use IBM Secure Execution, you must have host keys for your host machine(s) and they must be specified in your Ignition config. Addtionally, you must use LUKS encryption for root file system integrity.

For more information, see [insert xref to "Installing {op-system} using IBM Secure Execution"]
```
name of new file to reference to will be: `ibm-z-secure-execution.adoc`
Once I have the PR I'll let you know.

**Comment by jldohmann** [comment]
@SNiemann15 Do you think the RN should have an entry for this in the IBM Z and LinuxONE section too? I realize the content may be duplicated, but I imagine some people jump straight to certain sections that are relevant to them.

**Comment by jldohmann** [comment]
d38ef56..38fa000 implement Silke feedback

**Comment by SNiemann15** [comment]
> 
@jldohmann Thanks for bringing that up. 
We will surely add SE info to the IBM Z and LinuxONE section too. I need to discuss with @holgwolf if we only want to add SE to the list of supported features or add more info.

**Comment by SNiemann15** [comment]
@jldohmann I totally forgot to give you the PR for the IBM SE installation: https://github.com/openshift/openshift-docs/pull/52661

**Comment by jldohmann** [comment]
Thank you @SNiemann15 ! Were you and @holgwolf able to sync about what to add to the IBM Z and LinuxONE section of the RN? I'm happy to add it to this PR if so!

**Comment by SNiemann15** [comment]
@jldohmann Thanks for the offer we have it included in this PR: https://github.com/openshift/openshift-docs/pull/53464

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -44,6 +44,14 @@ This release adds improvements related to the following components and concepts.
...
+==== IBM Secure Execution on {ibmzProductName} and LinuxONE (Technology Preview)
+{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and LinuxONE (s390x architecture) as a Technology Preview feature. IBM Secure Execution provides the highest level of isolation and security for cluster workloads, and you can enable it by using a IBM Secure Execution-ready QCOW2 boot image.
+
+To use IBM Secure Execution, you must have host keys for your host machine(s) and they must be specified in your Ignition config. Addtionally, you must use LUKS encryption for root file system integrity.
```
Comment (holgwolf) [inline release_notes/ocp-4-12-release-notes.adoc:51]: LUKS encrypted volumes are automatically created during first boot. Therefore we should better mention
with Secure Execution you will automatically LUKS encrypt your boot volumes for higher Security.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 3]
```diff
@@ -44,6 +44,14 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-12-secure-execution-z-linux-one"]
+==== IBM Secure Execution on {ibmzProductName} and LinuxONE (Technology Preview)
+{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and LinuxONE (s390x architecture) as a Technology Preview feature. IBM Secure Execution provides the highest level of isolation and security for cluster workloads, and you can enable it by using a IBM Secure Execution-ready QCOW2 boot image.
```
Comment (holgwolf) [inline release_notes/ocp-4-12-release-notes.adoc:49]: Can we mention that IBM Secure Execution is a Hardware Enhancement and protects Memory Boundaries for KVM Guests?

Comment (jldohmann) [reply release_notes/ocp-4-12-release-notes.adoc]: For sure, added

File: release_notes/ocp-4-12-release-notes.adoc (hunk 2 above)
Comment (jldohmann) [reply release_notes/ocp-4-12-release-notes.adoc]: Thank you, updated

**Comment by jldohmann** [comment]
@holgwolf thank you for the feedback, I've made some updates. Could you PTAL for the lgtm? @mike-nguyen could you also PTAL for QE ack? Thank you!

File: release_notes/ocp-4-12-release-notes.adoc [hunk 4]
```diff
@@ -44,6 +44,14 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-12-secure-execution-z-linux-one"]
+==== IBM Secure Execution on {ibmzProductName} and LinuxONE (Technology Preview)
+{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and LinuxONE (s390x architecture) as a Technology Preview feature. IBM Secure Execution is a hardware enhancement and protects memory boundaries for KVM guests. IBM Secure Execution provides the highest level of isolation and security for cluster workloads, and you can enable it by using a IBM Secure Execution-ready QCOW2 boot image.
+
+To use IBM Secure Execution, you must have host keys for your host machine(s) and they must be specified in your Ignition config. IBM Secure Execution automatically encrypts your boot volumes using LUKS encryption.
+
+For more information, see xref:../installing/installing_ibm_z/installing-ibm-z-kvm_ibm-z-secure-execution.adoc[Installing {op-system} using IBM Secure Execution]
```
Comment (bscott-rh) [inline release_notes/ocp-4-12-release-notes.adoc:53]:
```suggestion
For more information, see xref:../installing/installing_ibm_z/installing-ibm-z-kvm_ibm-z-secure-execution.adoc[Installing {op-system} using IBM Secure Execution].
```

File: release_notes/ocp-4-12-release-notes.adoc [hunk 5]
```diff
@@ -44,6 +44,14 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-12-secure-execution-z-linux-one"]
+==== IBM Secure Execution on {ibmzProductName} and LinuxONE (Technology Preview)
+{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and LinuxONE (s390x architecture) as a Technology Preview feature. IBM Secure Execution is a hardware enhancement and protects memory boundaries for KVM guests. IBM Secure Execution provides the highest level of isolation and security for cluster workloads, and you can enable it by using a IBM Secure Execution-ready QCOW2 boot image.
+
+To use IBM Secure Execution, you must have host keys for your host machine(s) and they must be specified in your Ignition config. IBM Secure Execution automatically encrypts your boot volumes using LUKS encryption.
```
Comment (bscott-rh) [inline release_notes/ocp-4-12-release-notes.adoc:51]:
```suggestion
To use IBM Secure Execution, you must have host keys for your host machine(s) and they must be specified in your Ignition configuration files. IBM Secure Execution automatically encrypts your boot volumes using LUKS encryption.
```
expanding "config" per IBM style guide

**Review by bscott-rh — APPROVED** [review]
Couple of small nits/suggestions. I locally built and verified the preview. LGTM, noting that the xref is invalid until the feature docs are merged.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**